### ISSUE TITLE
Youtube has patched the unblocker?

### ISSUE THREAD

**Issue by hl2jeep** [zerodytrash/Simple-YouTube-Age-Restriction-Bypass#216]
### What happened?

Hi, i was trying to unblock some videos today when i noticed i just get a strange error, the plugin also encounters some errors, not sure whats going on but i have provided screenshots and logs

### Platform

Extension

### Version

2.5.4

### What browser are you on?

Firefox

### Browser version

117.0.1

### Link to the video

https://www.youtube.com/watch?v=UdHn-OFShTc

### Screenshots or Videos

[log.txt](https://github.com/zerodytrash/Simple-YouTube-Age-Restriction-Bypass/files/12703041/log.txt)
![image](https://github.com/zerodytrash/Simple-YouTube-Age-Restriction-Bypass/assets/51200953/73cdc8a7-e4a7-4a35-8bd8-4e249ba77bb6)

### About you

- [ ] I am a developer and/or have knowledge in programming

**Comment by hl2jeep** [comment]
i should probably add im using this in a private browsing tab.

**Comment by drunkwinter** [comment]
Duplicate of #197, #195, #191.

Hi, this issue was fixed in [v2.5.7](https://github.com/zerodytrash/Simple-YouTube-Age-Restriction-Bypass/releases/tag/v2.5.7) and later.

Unfortunately the extension got removed from Firefox Add-ons, so only installing the userscript is possible.
